ksqldb-streams/src/main/java/io/confluent/ksql/execution/streams/materialization/Row.java

```diff
@@ -80,6 +81,7 @@ public GenericKey key() {
   }
 
   @Override
+  @SuppressFBWarnings(value = "EI_EXPOSE_REP", justification = "should be mutable")
```

Comment:
If not mutable, `KsqlMaterializationTest` breaks. (Wondering is this is really intentional?)